packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -413,30 +401,31 @@ export class Nodes extends cdk.Construct {
     props.nodes.connections.allowInternally(new ec2.UdpAllPorts());
     props.nodes.connections.allowInternally(new ec2.IcmpAllTypesAndCodes());
 
-    // Cluster to:from rules
+    // Cluster to:nodes rules
     props.nodes.connections.allowFrom(this.cluster, new ec2.TcpPort(443));
-    props.nodes.connections.allowFrom(
-      this.cluster,
-      new ec2.TcpPortRange(1025, 65535)
-    );
+    props.nodes.connections.allowFrom(this.cluster, new ec2.TcpPortRange(1025, 65535));
+
+    // Allow HTTPS from Nodes to Cluster
+    props.nodes.connections.allowTo(this.cluster, new ec2.TcpPort(443));
+
+    // Allow all node outbound traffic
+    props.nodes.connections.allowToAnyIPv4(new ec2.TcpAllPorts());
```

Comment:
Hey heads up you might be thrashing on this bug: https://github.com/awslabs/aws-cdk/pull/998?

Also there is `new ec2.AllConnections()`?

Reply:
Yup, that's why those all outbound rules are set the way they are, all tcp,udp ports and ICMP specific. lf I use ne ec2.AllConnections() for the outbound rules it gets ignored and only the more specific ones, such as 443 to the API Server gets created.

Though technically it works for the AllowInternally() must've have just left there from testing. That makes sense to change as it's easier to read. Once the bug trashed, I'll modify the outbound ones.